src/graphics/texture.js

```diff
@@ -48,7 +48,9 @@ Object.assign(pc, function () {
      * @param {boolean} [options.mipmaps] - When enabled try to generate or use mipmaps for this texture. Default is true
      * @param {boolean} [options.cubemap] - Specifies whether the texture is to be a cubemap. Defaults to false.
      * @param {boolean} [options.volume] - Specifies whether the texture is to be a 3D volume (WebGL2 only). Defaults to false.
+     * @param {number} [options.type] - Specifies the image type, see {@link pc.TEXTURETYPE_DEFAULT}
      * @param {boolean} [options.rgbm] - Specifies whether the texture contains RGBM-encoded HDR data. Defaults to false.
+     * @param {boolean} [options.rgbe] - Specifies whther the texture contains RGBE-encoded HDR data. Defaults to false.
```

Comment:
Doesn't specifying the `type` property make specifying `rgbm` or `rgbe` redundant?

Reply:
Yes rgbe and swizzleGGGR flags are legacy.

Reply:
rgbm and swizzleGGGR has been deprecated.